sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/ReadFromKafkaDoFn.java

```diff
@@ -460,7 +460,28 @@ public ProcessContinuation processElement(
           }
           return ProcessContinuation.resume();
         }
+        long skippedRecords = 0L;
+        final Stopwatch sw = Stopwatch.createStarted();
```

Comment:
move this outside the while(true) loop
we're only going to be skipping at the beginning and we don't want the overhead per batch.

Reply:
Done.